### ISSUE TITLE
3D CAD Drawing  PID 4097

### ISSUE THREAD

**Issue by muttleytm** [adafruit/Adafruit_CAD_Parts#107]
I'd like to know the volume of the Adafruit  ADXL343 - Triple-Axis Accelerometer (+-2g/4g/8g/16g) w/ I2C/SPI - STEMMA QT / Qwiic.

I was told to look here on GIT hub and if there was to request one here.

Thanks

**Comment by muttleytm** [comment]
Thanks Noe,

I have another question. I am only familiar with STL files. In looking over
the different file types there are I see DAT files would most likely be the
3D CAD files, but there are a number of them.  I have tried opening a few
of them with some online viewers and also a linux viewer that is said to be
able to view them, but so far none of them have worked.

Can you tell me which specific file is the one equivalent to a STL file?
I'm guessing in the decision folder.

Thanks

Tom

**Comment by muttleytm** [comment]
Thanks, what viewer do I need to use to open it? I'm only familiar with stl files. I've looked around a little and I take it that the dat files are probably the ones, but there are quite a few of them.

I've downloaded and installed a linux viewer (LDviewer) but it doesn't work with any that I've tried. I've tried a few online viewers as well with no luck.  

I'm looking for a viewer capable of also figuring out the volume.
